walletkit/react-native/early-access/chain-abstraction.mdx

```diff
@@ -19,9 +19,9 @@ Apps need to pass `gas` as null when sending a transaction to allow proper gas e
 
 When sending a transaction, you need to:
 1. Check if the required chain has enough funds to complete the transaction
-2. If not, use the `prepare` method to generate necessary bridging transactions
-3. Check the status of the bridging transactions and wait for them to complete
-4. Once the bridging transactions are completed, execute the initial transaction
+2. If not, use the `prepareDetailed` method to generate necessary bridging transactions
```

Comment:
"detailed" shouldn't be developer-facing, that should just be an internal name. In the docs we should just need to refer to "prepare"

Reply:
Updated